src/engine/Algebra.ts

```diff
@@ -68,6 +68,15 @@ export class Vector {
       return true;
    }
 
+  /**
+  * Calculates distance between two Vectors
+  * @param vec1
+  * @param vec2
+  */
+  public static distanceBetweenVectors(vec1: Vector, vec2: Vector) {
```

Comment:
I think the indentation might be off? Also, let's just name this `distance`, it's obviously between Vector types 👍